apps/aehttp/test/aehttp_sc_SUITE.erl

```diff
@@ -1565,8 +1590,8 @@ sc_ws_contract_generic_(Origin, ContractSource, Fun, Config, Opts) ->
     [Fun(Owner, GetVolley, CreateContract, SenderConnPid,
          AckConnPid, OwnerPubkey, OtherPubkey, Opts, Config)
         || {Owner, {OwnerPubkey, OtherPubkey}} <- Actors],
-    ok = ?WS:unregister_test_for_channel_events(SenderConnPid, [sign, info, get, error]),
-    ok = ?WS:unregister_test_for_channel_events(AckConnPid, [sign, info, get, error]),
+    ok = ?WS:unregister_test_for_channel_events(SenderConnPid, EventTags),
```

Comment:
The function `with_registered_events(EventTags, [SenderConnPid, AckConnPid], fun() -> ... end)` could be used here.